js/modules.js

```diff
@@ -40,12 +46,13 @@ module.exports = (ENV, { MODULES, THIRDPARTY }) => {
         use: 'script-loader',
       },
       {
-        test: /\.modernizrrc$/,
-        use: [
-          'modernizr-loader',
-          'json-loader',
-        ],
+        test: /\.modernizrrc.js$/,
+        use: [ '@sect/modernizr-loader' ]
       },
+      {
+        test: /\.modernizrrc(\.json)?$/,
+        use: [ '@sect/modernizr-loader', 'json-loader' ]
+      }
```

Comment:
I think we should maybe aim to ditch modernizr. We don't use it all that much and it's a lot of extra work to add it to our pipline.